curvefs/src/client/common/config.h

```diff
@@ -70,8 +66,8 @@ struct ExtentManagerOption {
 };
 
 struct FuseClientOption {
-    MdsOption mdsOpt;
-    MetaServerOption metaOpt;
+    ::curve::client::MetaServerOption mdsOpt;
```

Comment:
//TODO
add using MdsOption = ::curve::client::MetaServerOption in curvefs/src/client/common/config.h

Reply:
done